installer/client/cli/fabric.py

```diff
@@ -75,11 +75,11 @@ def main():
                         help="Use Context file (context.md) to add context to your pattern", action="store_true")
 
     args = parser.parse_args()
-    home_holder = os.path.expanduser("~")
-    config = os.path.join(home_holder, ".config", "fabric")
-    config_patterns_directory = os.path.join(config, "patterns")
-    config_context = os.path.join(config, "context.md")
-    env_file = os.path.join(config, ".env")
+    home_holder = os.path.expanduser("~").replace("\\", "/")
+    config = os.path.join(home_holder, ".config", "fabric").replace("\\", "/")
```

Comment:
is this backward compatible with linux ?  os.path.join should handle OS-specific dir names

Reply:
after double checking, I was able to install from the same branch to both windows & linux using pipx without these modifications